scalafmt-interfaces/native/src/main/scala/org/scalafmt/interfaces/ScalafmtSession.scala

```diff
@@ -0,0 +1,26 @@
+package org.scalafmt.interfaces
```

Comment:
what's the difference with non native version? should we simply move from Java to scala in the original?